nat/traversal/pinger.go

```diff
@@ -50,8 +60,7 @@ type Pinger struct {
 	stopNATProxy   chan struct{}
 	once           sync.Once
 	natEventWaiter NatEventWaiter
```

Comment:
Can we call it with a constant case everywhere?
```suggestion
	natEventWaiter NATEventWaiter
```